tests/Operation/DistinctTest.php

```diff
@@ -42,6 +43,7 @@ public function testExplainableCommandDocument(): void
     {
         $options = [
             'collation' => ['locale' => 'fr'],
+            'hint' => '_id_',
```

Comment:
The `hint` option is supported in `explain` command on server version < 7.1.0?

Reply:
This is just a unit test, so we're only asserting that the option is included via the Explainable interface.